altering string for Calendar requests

xalgorithm opened the pull request.

greg-1-anderson commented:
LGTM, but re-add the composer.lock file to avoid problems with tests and deployment.

xalgorithm commented:
Ok, fixed it up. deleted bad branches


On Wed, May 30, 2018 at 4:17 PM, Greg Anderson <notifications@github.com>
wrote:

> LGTM, but re-add the composer.lock file to avoid problems with tests and
> deployment.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/westkingdom/website/pull/48#issuecomment-393349773>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AArGvudq0OT3m_O8mn637CKR6lMLjLb6ks5t3yiWgaJpZM4UUJmO>
> .
>



-- 


[image: --]

Vincent Flesouras
[image: https://]about.me/xalg
<https://about.me/xalg?promo=email_sig&utm_source=email_sig&utm_medium=email_sig&utm_campaign=external_links>
